CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-25
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Release notes 
+* Behavioral changes
+  * If your VM SKU does not support ephemeral or PremiumSSD OS disks, AKS will now use StandardSSD as the default OS disk type. Before, only VM SKUs that did not support ephemeral OS disks had this default.
+
+* Features
+  * Load Balancer TCP reset can now be desabled with a service annotaiton *"service.beta.kubernetes.io/azure-load-balancer-disable-tcp-reset=true"*. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4520) in preview
+  * Configurable resource group for the Prvate Link Service (PLS) creation using the *"ServiceAnnotationPLSResourceGroup = "service.beta.kubernetes.io/azure-pls-resource-group"* annotation. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4675).
+  * Support customizing probe config when *externalTrafficPolicy* = *local* and *health_probe_port* annotation is defined for service port. [upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4639)
+
+* Bug Fixes 
+  * A bug was fixed to improve the formatting of Kubernetes events such as cluster upgrade events.
+  * Fix for the possibility of duplicate podCIDR allocation when node PodCIDR patch request fails. [Upstream link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4568)
+  * Fix for the Private Link Service (PLS) creation failure that can occur if the customer selects a subnet name or PLS name that is too long. [Upstream Link](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/4538)
+
+* Component Updates
+  * Security Publisher image version has been updated to 1.0.67 from 1.0.64 which includes memory utilizaiton imrovements
```

Comment:
Nit:improvements .  Also link to the image version elaborating on the details of the type of improvements maybe useful.

Reply:
partially fixed, do not have  link